src/distances/delta.jl

```diff
@@ -1,9 +1,9 @@
 # Delta is not following the PreMetric rules since d(x, x) == 1
 struct Delta <: Distances.UnionPreMetric end
 
-(dist::Delta)(a::Number, b::Number) = a == b
+(dist::Delta)(a, b) = a == b
 Base.@propagate_inbounds function (dist::Delta)(
-    a::AbstractArray{<:Number}, b::AbstractArray{<:Number}
+    a::AbstractArray, b::AbstractArray
 )
```

Comment:
Formatter is unhappy here:
```suggestion
Base.@propagate_inbounds function (dist::Delta)(a::AbstractArray, b::AbstractArray)
```

Reply:
Yeah. Weird that it won't let me commit in the GH interface. Will make the change locally.